students/1st/feature:1st:hayounggam/js/login.js

```diff
@@ -0,0 +1,44 @@
+const idInput = document.querySelector("#js-id-input"),
+  pwInput = document.querySelector("#js-pw-input"),
+  btnLogin = document.querySelector(".btn-login");
+
+const GREEN_COLOR = "#00704a";
+const GREY_COLOR = "#dddddd";
+
+const toggleBtnLogin = () => {
+  const idLength = idInput.value.length;
+  const pwLength = pwInput.value.length;
+
+  idLength > 0 && pwLength > 0
+    ? (btnLogin.disabled = false)
+    : (btnLogin.disabled = true);
```

Comment:
javascript truthy, falsy에 대해서 검색해서 학습해보세요 :)
```suggestion
  btnLogin.disabled = !(idLength && pwLength)
```

Reply:
장현님 너무 감사합니다! 굳이 조건문을 이용하지 않아도 truthy, falsy를 이용하면 됐군요😱
이 부분 더 공부하겠습니다!🙇‍♀️

https://github.com/wecode-bootcamp-korea/webucks-javascript/pull/9/commits/387f3597f70f0db5e0b323f427a59f9c083e0ed1